app/models/physical_storage.rb

```diff
@@ -1,11 +1,13 @@
 class PhysicalStorage < ApplicationRecord
   include SupportsFeatureMixin
 
-  belongs_to :ext_management_system, :foreign_key => :ems_id, :inverse_of => :physical_storages,
-   :class_name => "ManageIQ::Providers::PhysicalInfraManager"
-  belongs_to :physical_rack, :foreign_key => :physical_rack_id, :inverse_of => :physical_storages
+  belongs_to :ext_management_system, :foreign_key => :ems_id
+  belongs_to :physical_rack, :foreign_key => :physical_rack_id
```

Comment:
The foreign key is not needed here as it matches the expected default.

Reply:
The `:physical_rack_id` was there when we got here (probably put there by Lenovo?). 
I'll remove it.

Reply:
Ah sorry...didn't notice it was previously there.... Thanks for fixing anyway!